src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,44 @@
 package core.basesyntax;
 
-class Pair {
+public class Pair<T1, T2> {
+    private final T1 item1;
+    private final T2 item2;
```

Comment:
why final?

Reply:
I know that final means that link will not be changed, but still don`t know where it`s supposed to be used...
Removed it as for now